test/generic/test_string.py

```diff
@@ -61,3 +62,8 @@ def test_string_methods():
     assert String("Hello %s") % "PySyft" == string
 
     assert str(string) == "Hello PySyft"
+
+    x = String("Hello PySyft")
+    bob = sy.VirtualWorker(id="bob", hook=sy.hook)
+    out = x.send(bob)
+    assert isinstance(out, sy.generic.pointers.string_pointer.StringPointer)
```

Comment:
This looks like a good thing to test! I might have missed how it's related to the rest of the PR though. Is it used here somewhere?

Reply:
It's not related - i was trying to figure out why code coverage was failing. I thought it might just be a rounding error or something so I tested an untested piece of code.